pyproject.toml

```diff
@@ -15,11 +15,39 @@ target-version = ["py36"]
 show_error_codes = true
 check_untyped_defs = true
 no_implicit_reexport = true
```

Comment:
Maybe there's logic for a `no_` option to work, but I think this should actually be:
`implicit_reexport = False`